config/ompi_check_psm2.m4

```diff
@@ -76,9 +80,7 @@ AC_DEFUN([OMPI_CHECK_PSM2],[
                ompi_check_psm2_happy="no"])])
 
         AS_IF([test "$ompi_check_psm2_happy" = "yes"],
-              [AS_IF([ test ! -z "$ompi_check_psm2_dir"],
```

Comment:
Won't removing these lines recreate the problem with PSM2 being installed in a non-default location?

Reply:
https://github.com/awlauria/ompi/commit/939129fc1ba805fdf2f90742f88244f609690e4b#diff-ee345127958f9c27de21699fdca72abb9e1f679ff2d6dd80fd7c0aa4e81755fbR83

Reply:
ops yes you're right.  i didn't do the merge conflict right.